syntax = "proto3";

package fixbug;

option cc_generic_services=true;

message ResultCode {
  int32 errcode=1;
  bytes errmsg=2;
}

message GetFriednsListRequest
{
    uint32 id = 1;
}

message GetFriednsListResponse
{
    ResultCode result = 1;
    repeated bytes friends=2;
}

service FriendServiceRpc
{
    rpc GetFriendList(GetFriednsListRequest) returns(GetFriednsListResponse);
}